cypress/support/pages/profile.PageObject.js

```diff
@@ -0,0 +1,15 @@
+import PageObject from '../PageObject';
+
+class ProfilePageObject extends PageObject {
+  url = '/profile/';
+
+  get usernameProfile() {
+    return cy.getByDataCy('profile-username');
+  }
+
+  get bioProfile() {
+    return cy.getByDataCy('profile-bio');
+  }
+}
+
+export default ProfilePageObject;
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better formatting and to follow best practices.

Reply:
added a newline